docs/user/README.md

```diff
@@ -1,29 +1,79 @@
 
 # Cloud Manager Module
 
+Use the Cloud Manager module to manage infrastructure providers' resources from a Kyma cluster.
+
 ## What is Cloud Manager?
 
-Cloud Manager is a central component that manages access to additional hyperscaler resources from the Kyma Runtime cluster. Its responsibility is to bring hyperscaler products/resources into the Kyma cluster in a secure way. Once Cloud Manager as a module is added to the Kyma cluster, Cloud Manager's features give you access to the respective products and resources of the infrastructure providers.
+Cloud Manager manages access to infrastructure providers' resources and products in SAP BTP, Kyma runtime. Once you add the Cloud Manager module to your Kyma cluster, it brings the resources in a secure way.
 
 ## Features
 
-Cloud Manager can provision the following cloud resources in the underlying cloud provider subscription:
+The Cloud Manager module provides the following feature:
 
-* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster
-* Virtual Private Cloud (VPC) Peering between your project and the Kyma cluster
+* Network File System (NFS) server that can be used as a ReadWriteMany (RWX) volume in the Kyma cluster.
+* Virtual Private Cloud (VPC) peering between your project and the Kyma cluster.
+* Google Cloud and Amazon Web Services Redis offering.
 
 ## Architecture
```

Comment:
Defines the basic terms, objects and entities of the service and describes their relationship to each other. A diagram is a really really nice to have :)